sktime/transformations/series/adapt.py

```diff
@@ -342,6 +349,10 @@ def _transform(self, X, y=None):
         else:
             Xt = self.transformer_.transform(**trafo_args)
 
+        # converting to dense if the transformer output was in sparse format
+        # Example: sklearn OneHotEncoder's default output is sparse
+        if isinstance(Xt, csr_matrix):
```

Comment:
is there a check that does not require the `scipy` dependency? E.g., get type name and compare to str

Reply:
Ah, I just tried it out and it worked, didn't know I could do that. Will change it.